awx/main/expect/isolated_manager.py

```diff
@@ -393,10 +335,18 @@ def run(self, instance, private_data_dir, playbook, module, module_args,
         self.ident = ident
         self.event_data_key = event_data_key
         self.instance = instance
-        self.host = instance.execution_node
         self.private_data_dir = private_data_dir
+        self.runner_params = self.build_runner_params(
+            [instance.execution_node],
+            verbosity=min(5, self.instance.verbosity)
+        )
         status, rc = self.dispatch(playbook, module, module_args)
         if status == 'successful':
             status, rc = self.check()
+        else:
```

Comment:
If the initial controller -> isolated dispatch fails, send an EOF event so the UI is happy.